src/image/components/runtime.tsx

```diff
@@ -72,7 +73,7 @@ export const Runtime: React.FC<IProps> = ({ authoredState }) => {
     <div className={css.runtime}>
       <div className={`${css.imageContainer} ${getImageLayout()}`} onClick={handleClick}>
         <img
-          src={url}
+          src={defaultImageUrl}
```

Comment:
```suggestion
          src={url || highResUrl}
```
The `defaultImageUrl` intermediate value doesn't seem to provide much value.

Reply:
@kswenson Thanks -- you're right on both counts. I'm going to be taking a different approach though in light of a new understanding about what the PT story is asking for.